source/opengl/runtime_gl.cpp

```diff
@@ -109,10 +112,25 @@ reshade::opengl::runtime_gl::runtime_gl()
 		config.get("OPENGL", "ReserveTextureNames", num_reserve_texture_names);
 		_reserved_texture_names.resize(num_reserve_texture_names);
 		config.get("OPENGL", "ForceMainDepthBuffer", _force_main_depth_buffer);
+		config.get("OPENGL", "IsUpscaled", _isUpscaled);
+		config.get("OPENGL", "AltDetection", _altDetection);
+		config.get("OPENGL", "SelectedDepthBuffer", _selectedDepthBuffer);
+		config.get("OPENGL", "AppName", _AppName);
+		config.get("OPENGL", "AppMode", _AppMode);
+		config.get("OPENGL", "AppWidth", _AppWidth);
+		config.get("OPENGL", "AppHeight", _AppHeight);
 	});
 	subscribe_to_save_config([this](ini_file &config) {
 		config.set("OPENGL", "ForceMainDepthBuffer", _force_main_depth_buffer);
+		config.set("OPENGL", "IsUpscaled", _isUpscaled);
+		config.set("OPENGL", "AltDetection", _altDetection);
+		config.set("OPENGL", "SelectedDepthBuffer", _selectedDepthBuffer);
+		config.set("OPENGL", "AppName", _AppName);
+		config.set("OPENGL", "AppMode", _AppMode);
+		config.set("OPENGL", "AppWidth", _AppWidth);
+		config.set("OPENGL", "AppHeight", _AppHeight);
 	});
+	setAppName(_AppName);
```

Comment:
I really don't want any application specific code in ReShade. Anything that is needed for a specific application could become necessary somewhere else too, so it should always be generalized.